Refresh.GameServer/Extensions/RequestContextExtensions.cs

```diff
@@ -18,4 +18,10 @@ public static (int, int) GetPageData(this RequestContext context, bool api = fal
 
         return (skip, count);
     }
+
+    [Pure]
+    public static bool IsPSP(this RequestContext context)
+    {
+        return context.RequestHeaders.Get("User-Agent") == "LBPPSP CLIENT";
+    }
```

Comment:
In general we tend to use arrow methods for small helper functions:

```suggestion
    [Pure]
    public static bool IsPSP(this RequestContext context) => context.RequestHeaders.Get("User-Agent") == "LBPPSP CLIENT";
```